.pre-commit-config.yaml

```diff
@@ -193,7 +151,7 @@ repos:
     language_version: python3
     additional_dependencies:
     - darglint
-    - flake8-docstrings
+    - flake8-docstrings # uses pydocstyle
```

Comment:
That's sorta the only thing it does. Why document it?

Reply:
Because not everyone knows that pydocstyle is use in that project if you cannot even find any matches using `pydocstyle` as a search string. I would have not added the comment if the plugin name was named `flake8-pydocstyle`.